lib/dynflow/rails/configuration.rb

```diff
@@ -96,7 +96,11 @@ def rake_task_with_executor?
       end
 
       def increase_db_pool_size?
-        !::Rails.env.test? && !remote?
+        !::Rails.env.test? && !remote? || sidekiq_worker?
```

Comment:
Should we add parens to make the order clearer?    like i assume its supposed to be:

```
!::Rails.env.test? && (!remote? || sidekiq_worker?)
```

Reply:
Ah, good point